Blueprints/BlueprintDefinitions/vs2022/MessageProcessingFramework/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -20,13 +20,95 @@ The framework reduces the amount of boiler-plate code developers need to write,
 
 The framework supports Open Telemetry via the [AWS.Messaging.Telemetry.OpenTelemetry](https://www.nuget.org/packages/AWS.Messaging.Telemetry.OpenTelemetry/) package. Refer to its [README](https://github.com/awslabs/aws-dotnet-messaging/blob/main/src/AWS.Messaging.Telemetry.OpenTelemetry/README.md) to enable instrumentation.
 
-## Testing Locally
+## Local Testing Guide
 
-The functions can be tested with the [Mock Lambda Test Tool](https://github.com/aws/aws-lambda-dotnet/tree/master/Tools/LambdaTestTool) in Visual Studio or other IDEs.
+### Prerequisites
+The functions can be tested with the [Mock Lambda Test Tool](https://github.com/aws/aws-lambda-dotnet/tree/master/Tools/LambdaTestTool-v2).
```

Comment:
nit: drop the word "Mock". Also be consistent when you refer to the tool. I see 4 different names: `Mock Lambda Test Tool`, `test tool`, `AWS Lambda Test Tool` and `Lambda Test Tool`